resourcemanager/schedule.go

```diff
@@ -31,13 +31,15 @@ func (r *ResourceManager) schedule() {
 }
 
 func (r *ResourceManager) schedulePool(pool *util.PoolContainer) scheduler.Command {
-	for _, sch := range r.scheduler {
-		cmd := sch.Tune(pool.Component, pool.Pool)
-		switch cmd {
-		case scheduler.Hold:
-			continue
-		default:
-			return cmd
+	if pool.Pool.Running() != 0 {
```

Comment:
Only a style suggestion:

```
if pool.Pool.Running() == 0 {
    return scheduler.Hold
}

xxxxx
```

But this one is also fine, not a big problem.

Reply:
Done